File: Data Structures & Algorithms/palindromic-substrings/submission-3.js
class Solution {
    /**
     * @param {string} s
     * @return {number}
     */
    countSubstrings(s) {
        function isPalindromic(s){
            let j = s.length ;
            if(j<=1) return true ;
            let i=0;
            j--;
            while(i<j){
                if(s[i]!=s[j])
                   return false ;
                i++;
                j--;
            }
            return true ;
        }
        let n = s.length;
        if(n<=1) return n ;
        let count = 0 ;
        for(let i=0;i<n;i++)
           for(let j = i+1 ;j<=n;j++)
             if(isPalindromic(s.substring(i,j)))
                 count++;
        return count ;
    }
}

File: Data Structures & Algorithms/guess-number-higher-or-lower/submission-3.py
# The guess API is already defined for you.
# @param num, your guess
# @return -1 if num is higher than the picked number
#          1 if num is lower than the picked number
#          otherwise return 0
# def guess(num: int) -> int:

class Solution:
    def guessNumber(self, n: int) -> int:
        left,right=1,n
        m=(left+right)//2
        i=guess(m)
        while i!=0 :
            if i==1:
                left=m+1
            else:
                right=m-1
            m=(left+right)//2
            i=guess(m)
        return m